pipelines/pgbench/benchmark/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM alpine:3.8
```

Comment:
if we use the docker action, then we don't need to define an action for this, i.e. no need to have a `benchmark/` folder. The `benchmark.sh` file could be placed in the `pgbench` parent folder